pkg/util/flags/flags.go

```diff
@@ -81,6 +82,14 @@ func createRequiredFlagError(name string, flags []string) error {
 	return fmt.Errorf("%s are required for %s", strings.Join(flags, ","), name)
 }
 
+func SetFlagsFromCmdline(flags *pflag.FlagSet) {
+	//Add commandline flags to the flagset. We will always write to stderr
+	//and not to a fle by default
+	flags.AddGoFlagSet(flag.CommandLine)
+	flags.Set("logtostderr", "true")
```

Comment:
will this affect only the `glog`, or also the `capnslog` logging?

Reply:
capnslog should not be affected by that. capnslog was already streaming the data to stdout if i am right. this change just moves the glog output from a tmp file to the std streams.